Makefile

```diff
@@ -11,4 +11,15 @@ python-test-%:
 	clear
 	python test_tooltool.py $*
 
-.PHONY: check shell-tests python-tests python-tests-%
+validate: _relengapi
+	source _relengapi/bin/activate && sh validate.sh
+
+PACKAGES = relengapi pep8 pyflakes coverage mock nose
+_relengapi:
+	virtualenv _relengapi
+	source _relengapi/bin/activate && pip install $(PACKAGES)
```

Comment:
Rather than creating a virtualenv for me, I think it would be better to just require an active virtualenv (`$VIRTUAL_ENV` set) when running validate -- in fact, validate already fails in this condition.  Everyone has different preferences for virtualenv names (I use `sandbox`).  Also, this won't automatically update the virtualenv.  Finally, `relengapi` isn't actually required -- we just need the virtualenv for pep8 etc.